rulesets/java/jpinpoint-rules.xml

```diff
@@ -22,9 +22,6 @@
         (jpinpoint-rules)</description>
         <priority>1</priority>
         <properties>
-            <property name="tag" value="jpinpoint-rule" type="String" description="for-sonar"/>
-            <property name="tag" value="performance" type="String" description="for-sonar"/>
-            <property name="version" value="2.0"/>
```

Comment:
Why is version disappearing? For PMD 6, version should be in there, for PMD 7 it should be removed.
For generating the jpinpoint-rules-pmd7 file, this is assumed.

Reply:
Version removal was already done manually in the classification branch, is not done by the merge script. I was not aware that it must be present in PMD6.

Reply:
added back the version for pmd-6